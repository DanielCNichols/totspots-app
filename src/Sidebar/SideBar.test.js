import React from 'react';
import ReactDOM from 'react-dom';
import SideBar from './SideBar.js';
import renderer from 'react-test-renderer';

it('renders SideBar without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<SideBar />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders SideBar as expected', () => {
  const tree = renderer.create(<SideBar name='SideBar' />).toJSON();
  expect(tree).toMatchSnapshot();
});
