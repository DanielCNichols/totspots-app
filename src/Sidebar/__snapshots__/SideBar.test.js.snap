// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SideBar as expected 1`] = `
<ul
  className="sidebar"
/>
`;
